Ghidra/Processors/SuperH4/data/languages/SuperH4.sinc

```diff
@@ -2374,8 +2374,8 @@ define pcodeop mac_wOp;
 
 	temp:4 = -M_0t;
 	N_0t = temp - zext($(T_FLAG));
-	$(T_FLAG) = ( 0 < temp );
-	if ( temp >= N_0t ) goto <skip>;
+	$(T_FLAG) = ( 0 s< temp );
```

Comment:
According to the manual, temp is an unsigned long, so these two checks should be unsigned.

Reply:
Ah, okay... There was just a warning there.  so if a change were made, maybe `temp != 0`

Reply:
That would be a solid improvement.